utils/wui.py

```diff
@@ -0,0 +1,220 @@
+#!/usr/bin/env python3
+
+import os
+from pathlib import Path
+import gzip
+import shutil
+
+class WUIFiles:
```

Comment:
/question/ Should this functionality be really associated with WUI itself? It seems to me, that it only takes some files, compresses them, and wraps/encodes them into C source files. It is a generic task that can be used by any other part of the firmware, not just WUI.

Reply:
This script is made specifically for generating fsdata_wui_local.c file, that have some static code parts just for handling WUI files. I know what you mean, and generating raw data from files could be separated and used for different tasks (for example - image assets from png files for fw GUI).